src/core/plugin-loader.cpp

```diff
@@ -84,7 +84,7 @@ std::pair<void*, void*> wf::get_new_instance_handle(const std::string& path)
     void *handle = dlopen(path.c_str(), RTLD_NOW | RTLD_GLOBAL);
     if (handle == NULL)
     {
-        LOGE("error loading plugin: ", dlerror());
+        LOGE("error loading plugin [", path, "]: ", dlerror());
```

Comment:
This appears to be the only change that the PR advertises. I'm not sure what the other changes are about, but they don't seem that they should be a part of this pull request.

Reply:
That is correct. That was the first commit and PR of four days ago.
After that I made two more PR requests, but there is no way to make separate PR's. It merges everything into this one. I thought that soreau told me to just keep adding stuff, maybe I misunderstood?

Reply:
The PR should be about an idea. If a commit is not part of the idea, it should go into a separate PR. If you have one PR that depends on another, you can submit the first one and then in the subsequent PRs, put a message that it depends on the first PR (number). Of course the dependent PRs won't build in CI until the first one is merged, but after it is, we can simply rerun CI and most times, you will not need to take further action.